main.go

```diff
@@ -0,0 +1,105 @@
+package main
+
+import (
+    "database/sql"
+    "fmt"
+
+    _ "github.com/lib/pq"
+)
+
+
+type Config struct {
+    Host     string 
+    Port     string 
+    Username string 
+    Password string 
+    Database string 
+}
+
+
+type Reader struct {
+    ReaderNum   int    `json:"reader_num"`
+    ReaderName  string `json:"reader_name"`
+    ReaderAdress string `json:"reader_adress"`
+    ReaderPhone string `json:"reader_phone"`
+}
+
+
+func NewDatabaseConnection(cfg Config) (*sql.DB, error) {
+    connectionString := fmt.Sprintf("host=%s port=%s user=%s password=%s dbname=%s sslmode=disable",
+        cfg.Host, cfg.Port, cfg.Username, cfg.Password, cfg.Database)
+    
+    db, err := sql.Open("postgres", connectionString)
+    if err != nil {
+        return nil, fmt.Errorf("ошибка при открытии соединения с БД: %v", err)
+    }
+    
+    err = db.Ping()
+    if err != nil {
+        return nil, fmt.Errorf("ошибка при проверке подключения к БД: %v", err)
+    }
+    
+    return db, nil
+}
+
+func GetReader(db *sql.DB, readerNum int) (*Reader, error) {
+    query := "SELECT reader_num, reader_name, reader_adress, reader_phone FROM readers WHERE reader_num = $1"
+    
+    row := db.QueryRow(query, readerNum)
+    reader := &Reader{}
+    
+    err := row.Scan(&reader.ReaderNum, &reader.ReaderName, &reader.ReaderAdress, &reader.ReaderPhone)
```

Comment:
Лучше делать так, выглядит более читаемо
```suggestion
    err := row.Scan(
        &reader.ReaderNum, 
        &reader.ReaderName, 
        &reader.ReaderAdress, 
        &reader.ReaderPhone,
        )
```